vital/config/config_block.h

```diff
@@ -573,11 +573,11 @@ config_block_get_value_cast( config_block_value_t const& value )
   std::stringstream str;
   str << value;
 
-  timestamp::time_t t;
+  kwiver::vital::time_us_t t;
```

Comment:
(Pedantic) Not sure this should be qualified? (Doesn't hurt, hence the "pedantic" label, but in general we seem to omit qualification for things in the current namespace...)

Likewise below...